### ISSUE TITLE
Next release?

### ISSUE THREAD

**Issue by gabeng** [etal/cnvkit#877]
Hi,

a few changes have accumulated on the master branch that would be very helpful. Are there any plans for a release in the near future?

**Comment by etal** [comment]
@gabeng  I'm aiming for one this week, while I have some time off.

**Comment by serge2016** [comment]
@etal thank you! Waiting for it!

**Comment by etal** [comment]
Folks, CNVkit 0.9.11 is released. Enjoy!

**Comment by serge2016** [comment]
Dear @etal! Could you add this tag https://github.com/etal/cnvkit/releases/tag/v0.9.11 to the releases https://github.com/etal/cnvkit/releases , please?

**Comment by etal** [comment]
Yep, freshly rolled: https://github.com/etal/cnvkit/releases/tag/v0.9.11

**Comment by serge2016** [comment]
Thank you very much!

**Comment by gabeng** [comment]
Apologies for my ignorance - what has to be done such that the bioconda recipe build job succeeds?
https://github.com/bioconda/bioconda-recipes/pull/47204

**Comment by etal** [comment]
This is a new one to me, looks like a static check on the conda packaging failed:
* https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=56108&view=logs&j=eb1ca712-fec5-5d93-deb0-40983fe61f2f&t=22a463be-d34e-5e0b-a210-c6720f3b9b60&l=32

Error message:

> ERROR: recipes/cnvkit/meta.yaml:0: missing_run_exports: Recipe should have a run_exports statement that ensures correct pinning in downstream packages

Explanation:
* https://bioconda.github.io/contributor/linting.html#missing-run-exports

I'll see what I can do.

**Comment by etal** [comment]
Here's a possible fix for the conda recipe:
https://github.com/etal/cnvkit/pull/880

I have no idea how to test whether this patch will fix the build for bioconda.

**Comment by gabeng** [comment]
Someone seems to have pushed it through. The docker build succeeded, too. All good from my point of view now.
